src/lib/rate-limit-shield-rule.ts

```diff
@@ -1,6 +1,6 @@
 import { GraphQLResolveInfo } from 'graphql';
-import { rule } from 'graphql-shield';
-import { Rule } from 'graphql-shield/dist/rules';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Instead of writing this exception, specify the module in the package.json under peerDepencies:

```
  "peerDependencies": {
    "graphql-shield": ">=123",
 ```